tests/python/direct_utils/utils.py

```diff
@@ -28,6 +28,11 @@ def is_pre_blackwell():
     return prop.major < 10
 
 
+def is_pre_blackwell_12():
```

Comment:
FYI, I just added `microarchitecture_is_pre` which you can replace is_pre_blackwell_12 with. Having both `blackwell` and `12` in the name is redundant.

Reply:
done.